flaml/model.py

```diff
@@ -667,6 +667,7 @@ def on_epoch_end(self, args, state, control, **callback_kwargs):
                     control.should_save = True
                     control.should_evaluate = True
 
+        self._training_args.report_to = []

```

Comment:
Please add comment why this line is needed.

Reply:
Removed